tests/subsys/logging/log_syst/src/main.c

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Test log message
+ */
+
+#include <zephyr.h>
+#include <logging/log.h>
+#include "mock_backend.h"
+#include <sys/printk.h>
+#include <logging/log_backend.h>
+#include <logging/log_backend_std.h>
+#include <logging/log_ctrl.h>
+#include <logging/log_output.h>
+#include <tc_util.h>
+#include <stdbool.h>
+#include <ztest.h>
+#include <stdlib.h>
+
+#define TEST_MSG_0 "0 args"
+#define PAYLOAD_MULTIPLE_ARGS "4465627567206D657373616765206578616D706C652C2025642C2025642C20256"\
```

Comment:
any chance of having a comment that would somehow explain this stream?
If it's STP stream maybe something like
`0x4 (D8) 0x46 0x5 (D16) ...` (not sure if i got it correct).

Reply:
@nordic-krch I have added the comment for PAYLOAD_MULTIPLE_ARGS explaining its purpose.